wolfssl/wolfcrypt/misc.h

```diff
@@ -39,6 +39,9 @@ word32 rotlFixed(word32, word32);
 WOLFSSL_LOCAL
 word32 rotrFixed(word32, word32);
 
+WOLFSSL_LOCAL
+word16 ByteReverseWord16(word16);
```

Comment:
What is this for?

Reply:
This is for wolfTPM, but since we moved it to a sep repo this is not required. I will remove it.